migtests/tests/analyze-schema/summary.json

```diff
@@ -8,6 +8,7 @@
             {
                 "ObjectType": "SCHEMA",
                 "TotalCount": 1,
+                "InvalidCount": 1,
```

Comment:
What is the logic for determining when a SCHEMA is invalid?

Reply:
it is this Create schema with elements regex case - 
```
CREATE SCHEMA hollywood
    CREATE TABLE films (title text, release date, awards text[])
    CREATE VIEW winners AS
        SELECT title, release FROM films WHERE awards IS NOT NULL;

createSchemaRegex.MatchString(sqlInfo.stmt) {
			summaryMap["SCHEMA"].invalidCount[sqlInfo.objName] = true
			reportCase(fpath, "CREATE SCHEMA with elements not supported yet.",
```